what4/test/OnlineSolverTest.hs

```diff
@@ -371,8 +373,9 @@ timeoutTests testLevel solvers =
       -- developer as solvers, What4 formulation, and machine speeds
       -- evolve.
       approxTestTimes :: [ (SolverName, Time) ]
-      approxTestTimes = [ (SolverName "Z3",         2.27 % Second)    -- Z3 4.8.10.  Z3 is good at self timeout.
+      approxTestTimes = [ (SolverName "Z3",         2.7 % Second)    -- Z3 4.8.10.  Z3 is good at self timeout.
```

Comment:
Why was this changed?

Reply:
Good catch, I was playing with the numbers and changed it back to the wrong time.